CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 0.14.0 TBD
 
+#### Changes
+- Relaxed rules for identifiers created via `Ident::new`, `ProcedureName::new`,
+  `LibraryNamespace::new`, and `Library::new_from_components`
+- [BREAKING] Renamed `Ident::new_unchecked` and `ProcedureName::new_unchecked`
+  to `from_raw_parts`
```

Comment:
As discussed offline, we could probably merge this into `main` and do a v0.13.1 patch release.

Reply:
Good point, I'll move this to the appropriate section of the CHANGELOG.